<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEntry</class>
 <widget class="QDialog" name="NewEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Entry</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="dialogButtonLayout">
    <item>
     <spacer name="dialogButtonSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="sermonFields_gridLayout">
    <item row="2" column="3">
     <widget class="QLineEdit" name="location_lineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="date_lbl">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="title_lineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="title_lbl">
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="speaker_lbl">
      <property name="text">
       <string>Speaker:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="location_lbl">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="description_lbl">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="description_lineEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="speaker_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="importAudio_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>21</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Audio:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>48</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="importAudioFiles_readOnlyEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>211</width>
     <height>73</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Please select one or more audio files for this sermon . . .</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
